cli/flags.rs

```diff
@@ -105,6 +106,11 @@ fn add_run_args<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
         .long("allow-run")
         .help("Allow running subprocesses"),
     )
+    .arg(
+      Arg::with_name("allow-native")
```

Comment:
`--allow-native` vs `--allow-plugin` 🤔 WDYT?

Reply:
The main function added to ts/js is `openPlugin`, so maybe `--allow-plugin`.